test/export-api-def.test.js

```diff
@@ -102,6 +104,25 @@ describe('loopback:export-api-def generator', function() {
     });
   });
 
+  describe('running on project with onging async operations', function() {
+    var PROCESS_EXIT_STUB;
+
+    before(function stubProcessExit() {
+      PROCESS_EXIT_STUB = sinon.stub(process, 'exit');
+    });
+
+    it('does not hang', function() {
```

Comment:
Since this test is not actually inspecting an app that would hang, I find the current test name misleading. I am proposing to rename it to something like "it eventually exits the process".